_rules/css-orientation-lock-media-queries.md

```diff
@@ -0,0 +1,156 @@
+---
+name: Page has no orientation lock specified using CSS media queries
+
+rule_type: atomic
+
+description: |
+  This rule checks that page content is not locked to any specific display orientation using CSS media queries, and the content is operable in all display orientations.
+
+success_criterion:
+- 1.3.4 # (Orientation)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any `element` that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element does not have any `CSS` media queries applied, that locks the page content to a specific orientation.
+
+## Assumptions
+
+- The rule does not take consider existence of any control on the page that can handle or change the orientation on demand.
+
+- The rule assumes that a specific display orientation is not [essential](https://www.w3.org/TR/WCAG21/#dfn-essential), which is listed as a valid exception to [Success Criterion 1.3.4 Orientation]((https://www.w3.org/WAI/WCAG21/Understanding/orientation.html)). Examples where a particular display orientation may be essential are a bank check, a piano application, slides for a projector or television, or virtual reality content where binary display orientation is not applicable.
```

Comment:
So, while a pass is a pass, this rule could also return a fail for something that would not be a fail of the WCAG guideline. "Valid exceptions" do make testing a little tricky, and for this rule not passing might require a manual check warning or something similar.

Reply:
Yeah, I also think this assumption is a bit too restrictive and should really make it's way into the expectation, i.e. if an orientation lock _is_ used then it _must_ be essential.